### PR TITLE
completed api-registration | Daniel Hong

### PR DESCRIPTION
**Pull request by daniel-sj-hong**
Capture
Capture1

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
the client should not receive the hashed password
image

**Comment by daniel-sj-hong** [comment]
Capture

**Comment by daniel-sj-hong** [comment]
Capture

File: api-registration/server/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      const sql = `
+     insert into "users" ("username", "hashedPassword")
+     values ($1, $2)
+     returning "userId", "username", "createdAt";
+     `;
+      const params = [username, hashedPassword];
+      db.query(sql, params)
+        .then(result => {
+          const newPassword = result.rows;
+          res.status(201).json(newPassword);
```
Comment (thebearingedge) [inline api-registration/server/index.js:40]: The client should receive an object and not an array.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
super-duper close!!!

**Comment by daniel-sj-hong** [comment]
Capture
Capture1

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**